services/ResponseCode.proto

```diff
@@ -218,5 +218,9 @@ enum ResponseCodeEnum {
   UNPARSEABLE_SCHEDULED_TRANSACTION = 207; // The bytes allegedly representing a transaction to be scheduled could not be parsed
   UNSCHEDULABLE_TRANSACTION = 208; // ScheduleCreate and ScheduleSign transactions cannot be scheduled
   SOME_SIGNATURES_WERE_INVALID = 209; // At least one of the signatures in the provided sig map did not represent a valid signature for any required signer
-  TRANSACTION_ID_FIELD_NOT_ALLOWED = 210; // The <tt>scheduled</tt> and <tt>nonce</tt> fields in the <tt>TransactionID</tt> may not be set in a top-level transaction
+  TRANSACTION_ID_FIELD_NOT_ALLOWED = 210; // The scheduled field in the TransactionID may not be set to true
+  IDENTICAL_SCHEDULE_ALREADY_CREATED = 211; // A schedule already exists with the same identifying fields of an attempted ScheduleCreate (that is, all fields other than scheduledPayerAccountID)
+  UTF8_ENCODING_CONTAINS_ZERO_BYTE = 212; // A string field in the transaction has a UTF-8 encoding with the prohibited zero byte
```

Comment:
Perhaps just `STRING_CONTAINS_ZERO_BYTE` or `INVALID_ZERO_BYTE_IN_STRING`? All proto strings are utf-8 encoding so that's assumed. What's important that it is a field of type string.

Reply:
Used `INVALID_ZERO_BYTE_IN_STRING`.